playground/nextjs/pages/_app.tsx

```diff
@@ -15,6 +15,8 @@ if (typeof window !== 'undefined') {
         },
         debug: true,
         __preview_send_client_session_params: true,
+        __preview_measure_pageview_stats: true,
+        scroll_root_selector: ['#scroll_element', 'html'],
```

Comment:
Quick note that you can't use the css comma here, i.e. `#scroll_element, html`. It would also return the html root element, event if `#scroll_element` is present on the page :)